org/MavericSystems/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: org
-title: Maveric Systems 
+title: MavericSystems 
```

Comment:
This one didn't actually have to be changed, just the one that's used to build the URL. But it won't hurt anything that you did.